modules/data-entry/src/main/frontend/src/questionnaire/QuestionnaireStyle.jsx

```diff
@@ -233,6 +233,31 @@ const questionnaireStyle = theme => ({
         },
       },
     },
+    timelinePaper: {
+        padding: "8px 16px",
+        marginBottom: "40px"
```

Comment:
The bubbles need to be pulled away a bit from the "duration" circles.
```suggestion
        padding: theme.spacing(1,2),
        marginBottom: theme.spacing(0, 1, 5),
```